viewing_party/party.py

```diff
@@ -0,0 +1,182 @@
+import statistics
+
+
+def create_movie(title, genre, rating):
+    """
+    Returns:
+        new_movie (dict): A dictionary initialized with title (str), genre (str), and rating (float).
+        Returns None if any parameters are falsy.
+    """
+
+    if title and genre and rating:
+        new_movie = {"title": title, "genre": genre, "rating": rating}
+        return new_movie
```

Comment:
Generally if I'm going to return it on the next line I don't assign to a variable:

```suggestion
        return {"title": title, "genre": genre, "rating": rating}
```